app/lint.xml

```diff
@@ -34,4 +34,7 @@
     <!-- We don't want to fail the build, nor do we want to to pollute our informational output. -->
     <issue id="Typos" severity="informational"/>
 
+    <!--  For debug manifest. This is only a temporary change.  -->
```

Comment:
nit: we may replace this with `tools:ignore`: see my comment below.

If we don't, can you elaborate why this occurs with the debug manifest? If it's a temporary change, we should probably add a bug # to explain why we can remove it